content/en/docs/tasks/administer-cluster/enabling-service-topology.md

```diff
@@ -0,0 +1,53 @@
+---
+reviewers:
+- andrewsykim
+- johnbelamaric
+- imroc
+title: Enabling Service Topology
+content_template: templates/task
+---
+
+{{% capture overview %}}
+This page provides an overview of enabling Service Topology in Kubernetes.
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+  {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Introduction
+
+_Service Topology_ enables a service to route traffic based upon the Node
+topology of the cluster. For example, a service can specify that traffic be
+preferentially routed to endpoints that are on the same Node as the client, or
+in the same availability zone.
+
+## Prerequisites
```

Comment:
I recommend moving this to the ` {{% capture prerequisites %}}` section (at time of writing, that starts on line 15)

Reply:
Good catch, thank you

Reply:
So the page actually renders the prerequisite capture as a "Before you begin"  so I think it looks okay as-is. 

https://5ddcbf5c6044c00008fa6e29--kubernetes-io-vnext-staging.netlify.com/docs/tasks/administer-cluster/enabling-service-topology/